README.md

```diff
@@ -8,55 +10,44 @@ This buildpack will participate if all the following conditions are met
 
-The buildpack will do the following for Java applications:
-
-* Contributes the OpenTelemetry Java agent to a layer and configures `$JAVA_TOOL_OPTIONS` to use it
-
-## Testing
+At build time, the buildpack will do the following for Java applications:
 
-Given a Java application, you can package it by running the following command from the current folder.
+* Contributes the OpenTelemetry Java agent to a layer and configures `$JAVA_TOOL_OPTIONS` to use it.
+* By default, the agent is configured to be disabled (`OTEL_JAVAAGENT_ENABLED=false`).
+* By default, the metrics exporting feature of the agent is configured to be disabled (`OTEL_METRICS_EXPORTER=none`).
 
-```shell
-pack build my-app \
-  --path <my_app_path> \
-  --buildpack paketo-buildpacks/java \
-  --buildpack . \
-  --builder paketobuildpacks/builder:base \
-  --verbose --trust-builder \
-  -e BP_JVM_VERSION=17 -e BP_OPENTELEMETRY_ENABLED=true
-```
+At run time, the buildpack will do the following for Java applications:
 
-Next, run it as follows.
+* If a binding with `type` of `opentelemetry` exists, it uses the config tree from the binding to configure the OpenTelemetry Java agent. This is recommended when the configuration involves secrets of any kind.
 
-```shell
-docker run --rm -p 8080:8080 my-app
-```
-
-Check the logs and verify the OpenTelemetry Agent was included.
+## Usage
 
-```log
-Picked up JAVA_TOOL_OPTIONS: -Djava.security.properties=/layers/paketo-buildpacks_bellsoft-liberica/java-security-properties/java-security.properties -XX:+ExitOnOutOfMemoryError -javaagent:/layers/paketo-buildpacks_opentelemetry/opentelemetry-java/opentelemetry-javaagent.jar -XX:ActiveProcessorCount=3 -XX:MaxDirectMemorySize=10M -Xmx3537385K -XX:MaxMetaspaceSize=98270K -XX:ReservedCodeCacheSize=240M -Xss1M -XX:+UnlockDiagnosticVMOptions -XX:NativeMemoryTracking=summary -XX:+PrintNMTStatistics
+Once you enable the OpenTelemetry buildpack at build-time, you can configure it at run-time via environment variables, as described in the [project documentation](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/) or by passing configuration properties to the container via a binding.
 
-[otel.javaagent 2022-06-28 21:16:29:365 +0000] [main] INFO io.opentelemetry.javaagent.tooling.VersionLogger - opentelemetry-javaagent - version: 1.15.0
-```
+By default, the following configuration is applied to the OpenTelemetry Java Agent at run time.
 
-## Usage
+* `OTEL_JAVAAGENT_ENABLED=false`
+* `OTEL_METRICS_EXPORTER=none`
 
-Once you enable the OpenTelemetry buildpack, you can configure it at run-time via environment variables, as described in the [project documentation](https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/).
+When using a binding to configure the agent, properties are expected to be passed via a config tree format,
+where each key is the name of a file and the value its content. Keys can follow the environment variable format or the
```

Comment:
Minor note. This might be too much information and distracting from the point of this section. A binding is expected to use the format being described here. If you want, you could maybe link to the [Paketo Docs on Bindings](https://paketo.io/docs/howto/configuration/#bindings). That explains the format more.

Reply:
I made it shorter based on your suggestion and added a link to the bindings docs.